py3status/modules/volume_status.py

```diff
@@ -149,6 +143,97 @@ def current_volume(self):
         # get info about channel mute status
         muted = self._get_muted(output)
 
+        return perc, muted
+
+    def volume_up(self, delta):
+        # volume up
+        cmd = 'amixer -q -D {} sset {} '.format(self.device, self.channel)
```

Comment:
True, to be fair `volume_up()` etc should probably just return a command to run maybe even change the name to `volume_up_cmd()`